Formula/visp.rb

```diff
@@ -0,0 +1,65 @@
+class Visp < Formula
+  desc "Visual Servoing Platform library"
+  homepage "https://visp.inria.fr"
+  url "https://gforge.inria.fr/frs/download.php/latestfile/475/visp-3.1.0.tar.gz"
+  sha256 "2a1df8195b06f9a057bd4c7d987697be2fdcc9d169e8d550fcf68e5d7f129d96"
+
+  depends_on "cmake"     => :build
```

Comment:
only one space between "cmake" and `=>` please

Reply:
ok